logstash-core/lib/logstash/api/lib/app/service.rb

```diff
@@ -40,4 +44,15 @@ def get(key)
     end
     LogStash::Json.dump(data)
   end
+
+  private
+
+  def has_counters?
```

Comment:
This was actually a source of concern to me, as for testing I do really need to wait till some events are generated. Obviously open to improve this part of the code.